qa/L0_grpc_state_cleanup/cleanup_test.py

```diff
@@ -79,7 +80,7 @@ def _prepare_inputs_and_outputs(self, kind):
             self.outputs_.append(grpcclient.InferRequestedOutput("OUT"))
             self.outputs_.append(grpcclient.InferRequestedOutput("IDX"))
             self.requested_outputs_ = self.outputs_
-        elif kind == "simple" or kind == "streaming":
+        elif kind in ("simple", "streaming"):
```

Comment:
Just curious, what is `streaming` compared to `decoupled_streaming` and `non_decoupled_streaming`?

Reply:
The `decoupled_streaming` and `non_decoupled_streaming` are for the `repeat_int32` and `repeat_int32_non_decoupled` models on repeat backend. The `simple` and `streaming` are for the `custom_zero_1_float32` model on identity backend.

I think we could do some refactoring on how the inputs are prepared, i.e. for the repeat backend, we can have a function that returns a `inputs` that can be used directly on the `client.async_stream_infer()`.